.github/workflows/prod-workflow.yml

```
@@ -127,6 +127,58 @@ jobs:
+          cargo binstall dioxus-cli --force
+          cargo binstall toml-cli --force
+      
+      - name: Install Tauri CLI
+        run: cargo install tauri-cli
+      
+      - name: Decode keystore
+        run: |
+          echo "$KEYSTORE_BASE64" | base64 -d > release.keystore
+        env:
+          KEYSTORE_BASE64: ${{ secrets.KEYSTORE_BASE64 }}
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)